modules/core/test/v2/unit/recovery.ts

```diff
@@ -357,8 +374,7 @@ describe('Recovery:', function() {
     }));
 
     it('should generate BCH recovery tx with KRS', co(function *() {
-      recoveryNocks.nockBchRecovery(bitgo, true);
-
+      nockCoingecko(1000,'bitcoin-cash');
```

Comment:
What's `KRS`? (sorry 😄 )

Reply:
Key recovery service, providers would hold the backup keys, instead of the customer themselves.

Reply:
no worries :) thank you so much for your thorough review!! @matthewzipkin-bitgo